nameko_sqlalchemy/__init__.py

```diff
@@ -0,0 +1,35 @@
+from nameko.extensions import DependencyProvider
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+ORM_DB_URIS_KEY = 'ORM_DB_URIS'
+
+
+class OrmSession(DependencyProvider):
+    def __init__(self, declarative_base):
+        self.declarative_base = declarative_base
+        self.sessions = {}
```

Comment:
Hoisted from the original implementation. We pop them out in ``worker_teardown`` which is a pretty good guarantee, but I can add